app/src/main/java/dev/arkbuilders/navigator/presentation/screen/gallery/galleryuplift/domain/ProgressWithText.kt

```diff
@@ -0,0 +1,6 @@
+package dev.arkbuilders.navigator.presentation.screen.gallery.galleryuplift.domain
+
+data class ProgressWithText(
+    val isVisible: Boolean,
+    val text: String,
+)
```

Comment:
I made a mistake. This object is created in the viewmodel, where we cannot use string resources, but use plain text. Perhaps it’s worth making an enum with different progress states and turning it into text in fragment.